index.html

```diff
@@ -9,16 +9,27 @@
 </head>
 
 <body>
+   <dialog class="modal-1">   
+      <div><span class="close-1">&times;</span></div>
+      <nav>
+        <ul class = "lnks">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+      
+  </dialog>
    <header class="toolbar">
       <div class="logo">
          <h3>My Logo</h3>
       </div>
 
       <nav>
-         <ul class="nav">
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contact</li>
+         <ul class="desktop-nav">
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
          </ul>
          <a href="#">
             <img src="./images/hamburger.svg" alt="hamburger menu" class="hamburger-img" id="hamburger">
```

Comment:
According to the Figma template, I suggest you stick to the design as much as possible (e.g., font, colours, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), currently your popup mobile menu and positioning do not match the required design.

> ## Your Design
>
><img width="375" alt="Screenshot 2023-06-26 at 11 49 40 AM" src="https://github.com/cloud007bond/Portfolio/assets/91237525/3fd4a70c-eec9-46ce-b2a6-47cab259c3a9">
>
> ## The Figma Design
>
><img width="375" alt="Screenshot 2023-06-26 at 12 03 48 PM" src="https://github.com/cloud007bond/Portfolio/assets/91237525/d17c9244-f11b-49b3-b173-aa8b6c589d1e">